### ISSUE TITLE
The loss function to use

### ISSUE THREAD

**Issue by qobilidop** [qobilidop/srcnn#1]
Are we going to use the same MSE loss function as in the paper? Shall we experiment with different loss functions?

**Comment by xiongliyang219** [comment]
Let's use MSE first, if we have time, we can try to others. I think the answer to each question is "if we have time"

**Comment by qobilidop** [comment]
@xiongliyang219 I agree we should try more things if we have time. I also think we should have some discussion on whether those things are worth trying. Some of them might already been tried and we can refer to the conclusions from the literatures. I think we should do this in the progress report.

**Comment by qobilidop** [comment]
How about [perceptual loss](https://arxiv.org/abs/1603.08155)?

**Comment by jiacong1990** [comment]
Good idea. The paper achieves "visually pleasing results" on super resolution. I'll check if there's a simple way to implement this perceptual loss in Keras.

**Comment by qobilidop** [comment]
Progress report is submitted.
